jenkinsbuilder/cccp_index_reader.py

```diff
@@ -24,8 +24,10 @@ def projectify(
     new_project[0]['project']['name'] = appid
     new_project[0]['project']['git_url'] = giturl
     new_project[0]['project']['git_branch'] = gitbranch
-    new_project[0]['project']['rel_path'] = ('/' + gitpath) \
-        if (gitpath and not gitpath.startswith('/')) else '/'
+    rel_path = "/"
+    if gitpath and gitpath != "/":
```

Comment:
shouldn't either of condition suffice for the operation performed ?
like
`if gitpath != "/":` 
since if gitpath is `None`, the prior check before `and` does not make sense.